HerPublicWebsite/Views/Questionnaire/Eligible.cshtml

```diff
@@ -0,0 +1,124 @@
+﻿@using GovUkDesignSystem
```

Comment:
I can't find the latest designs for this (the link on the ticket looks old), but I think it looks sensible - I guess you've checked with the design team.